### DISCUSSION TITLE
packages to be installed in the base enviroment

### DISCUSSION THREAD

**Discussion by begovegar** [PaddlePaddle/PaddleSpatial#156] [BaiduKDDCup2022] [answered]
Could you please install the following packages in the base enviroment?
pip install skforecast
pip install prophet (probably you need to install pip install pystan==2.19.1.1 before)
pip install sktime
pip install statsmodels

We use all of them to forecast all the variables we won't know in the next 2 days after the test_x of 14 days.
Thank you.

**Comment by ramber1836** [comment, marked as answer]
These pkgs have been added to the base environment.

**Reply by begovegar** [reply]
Ok, thanks!

**Reply by begovegar** [reply]
Could you please add math to the list, please?
